src/Symfony/Component/PasswordHasher/Hasher/PasswordHasherFactory.php

```diff
@@ -23,14 +23,11 @@
  */
 class PasswordHasherFactory implements PasswordHasherFactoryInterface
 {
-    private array $passwordHashers;
-
     /**
      * @param array<string, PasswordHasherInterface|array> $passwordHashers
      */
-    public function __construct(array $passwordHashers)
+    public function __construct(private array $passwordHashers)
```

Comment:
```suggestion
    public function __construct(
        private array $passwordHashers,
    )
```

New line if we use CPP please

Reply:
done